src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsServiceOptionsExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet.Extensions
+{
+    /// <summary>
+    /// Extensions for Application Insights Service Options that returns specific set of properties if serviceOptions is enabled. If not, it returns the default initial value.
+    /// </summary>
+    internal static class ApplicationInsightsServiceOptionsExtensions
+    {
+        private static ApplicationInsightsServiceOptions _serviceOptions = new ApplicationInsightsServiceOptions();
+
+        /// <summary>
+        /// Checks if sampling is disabled through service options.
+        /// </summary>
+        internal static bool GetDisableDefaultSampling(this ApplicationInsightsServiceOptions serviceOptions)
```

Comment:
Today it's just a boolean, but tomorrow it can be a complex option which needs some functionality being added from our side. PLease let me know if this makes sense.